packages/core/src/js/sdk.tsx

```diff
@@ -4,11 +4,12 @@ import {
   defaultStackParser,
   makeFetchTransport,
 } from '@sentry/react';
-import type { Integration, Scope,UserFeedback } from '@sentry/types';
+import type { Breadcrumb, BreadcrumbHint,Integration, Scope,UserFeedback  } from '@sentry/types';
```

Comment:
Suggestion for extra spacing.
```suggestion
import type { Breadcrumb, BreadcrumbHint, Integration, Scope, UserFeedback  } from '@sentry/types';
```

Reply:
Good catch 🙇